src/pages/Login/LoginStyled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import { theme } from '../../style/theme';
+
+export const ParentContainer = styled.div`

```

Comment:
Could the styled in these to styled components be merged?

Reply:
Well here i just need display and justify props from the parentcontainer, and need to size the width but yes i could merge this styled

Reply:
By the way Steven do you have any pattern to use with StyledComponent about naming or something related to it?